app/src/util/tools.ts

```diff
@@ -134,12 +145,16 @@ export const convertLocalToUTC = (date: Date): Date => {
 }
 
 export const divBN = (a: BigNumber, b: BigNumber, scale = 10000): number => {
-  return (
-    a
-      .mul(scale)
-      .div(b)
-      .toNumber() / scale
-  )
+  try {
+    return (
+      a
+        .mul(scale)
+        .div(b)
+        .toNumber() / scale
+    )
+  } catch {
+    return 999999999999999
+  }
```

Comment:
Why do we return this in the catch?

Reply:
Dunno how these things slipped up I will go through the code again...